lib/seed/Seeder.js

```diff
@@ -28,7 +20,7 @@ class Seeder {
   // Runs seed files for the given environment.
   async run(config) {
     this.config = this.setConfig(config);
-    const [all] = await this._seedData();
+    const all = await this._listAll();
```

Comment:
Will types changed in https://github.com/knex/knex/pull/3438/files still be correct after this change?

Reply:
_seedData is just _listAll, but wrapping result inside promise into []